server/webapp/WEB-INF/rails.new/app/views/admin/materials/pluggable_scm/show_existing.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= render(:partial => "shared/flash_message") %>
+<%= render :partial => "shared/config_save_actions" -%>
+
+<%= form_for(@material, :as => :material, :url => admin_pluggable_scm_choose_existing_path, :html => {"novalidate"=> "novalidate", :id=> "material_form", :name=> "material_form", :method => "POST", :onsubmit => "return AjaxForm.jquery_ajax_submit(this);", :class => "popup_form"}) do |f| %>
+    <%= md5_field %>
+
+    <div class="form_content exec_task_editor">
+        <%= render :partial => "admin/shared/global_errors.html", :locals => {:scope => {}} -%>
+        <div id="material">
+            <%= render :partial => 'shared/form_required_message.html', :locals => {:scope => {}} %>
+
+            <%- if @scms.empty? -%>
+                <p class="warnings">No SCMs found. Please add a SCM first.</p>
+            <%- end -%>
+
+            <div class="fieldset">
```

Comment:
I think we should allow editing with a warning about global entity & ask for feedback from users. Im ok removing/turning off code if others think otherwise.